CV Backend/src/main/java/com/qa/demo/tests/TestFunctions.java

```diff
@@ -0,0 +1,84 @@
+package com.qa.demo.tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class TestFunctions {
```

Comment:
this file seems to be in the testing bit of the project structure not in the main

Reply:
moved